WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsHubTimeRangeController.swift

```diff
@@ -0,0 +1,146 @@
+import Foundation
+import Yosemite
+
+struct TimeRange {
+    let start: Date
+    let end: Date
+}
+
+public class AnalyticsHubTimeRangeController {
+
+    private let currentTimezone = TimeZone.current
```

Comment:
Since we are storing the Timezone, I think it's worth using `.autoupdatingCurrent` to keep any user change.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8193/commits/f61047222656451277b695236307b96a3a235088